AnkiConnect.py

```diff
@@ -685,6 +686,26 @@ def guiDeckReview(self, name):
             return False
 
 
+    def setLock(self, timeout=0):
+        if hasattr(self, 'lockTimer'):
```

Comment:
Can you just make `lockTimer` be always be present on the object and be initialized to `None` instead? I